packages/react-drag-drop/src/components/DragDrop/DragDropSort.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import {
+  DndContext,
+  closestCenter,
+  DragOverlay,
+  DndContextProps,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent,
+  DragStartEvent
+} from '@dnd-kit/core';
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy
+} from '@dnd-kit/sortable';
+import { Draggable } from './Draggable';
+import { DraggableDataListItem } from './DraggableDataListItem';
+import { DraggableDualListSelectorListItem } from './DraggableDualListSelectorListItem';
+import { DraggableObject } from './DragDropUtil';
+import styles from '@patternfly/react-styles/css/components/DragDrop/drag-drop';
+
+export type DragDropSortDragEndEvent = DragEndEvent;
+export type DragDropSortDragStartEvent = DragStartEvent;
+
+export interface DragDropSortProps extends DndContextProps {
```

Comment:
The fact that this extends DndContextProps is good. There are a LOT there for accessibility and such that are good to expose to consumers: https://docs.dndkit.com/api-documentation/context-provider#props

I wonder if we should at least link to these dnd-kit docs in our docs (sort of like how we do it in our charts docs). I _think_ you might even be able to put a comment on line 29 above the interface and it will put that description above the generated props table? worth trying.

Reply:
Put a comment for the prop description but I think there's an issue with the docgen not picking up the props table?

Reply:
I'm seeing props tables on other docs pages. so there is something about the dragdrop docs that are causing these not to be generated...

Reply:
https://patternfly-react-pr-9784.surge.sh/components/wizard#wizard is an example of the Wizard props table being rendered correctly with a comment on the top of the props table.